x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/EnterpriseSearch.java

```diff
@@ -140,6 +151,12 @@ protected XPackLicenseState getLicenseState() {
             new ActionHandler<>(GetQueryRulesetAction.INSTANCE, TransportGetQueryRulesetAction.class),
             new ActionHandler<>(ListQueryRulesetsAction.INSTANCE, TransportListQueryRulesetsAction.class),
             new ActionHandler<>(PutQueryRulesetAction.INSTANCE, TransportPutQueryRulesetAction.class),
+
+            // Connectors
+            new ActionHandler<>(DeleteConnectorAction.INSTANCE, TransportDeleteConnectorAction.class),
```

Comment:
No GET?

Reply:
The logic of GET was slightly more complex than the list endpoint, so I left it out for future PRs. Carlos suggested that I just focus on adding a single, simple PUT endpoint as a part of this PR, and I will follow this approach.

Reply:
That's a good suggestion and the pattern we followed when adding APIs for search relevance 👍